api.py

```diff
@@ -52,6 +52,7 @@
     share_an_image,
     unshare_an_image,
 )
+from modules import script_callbacks
```

Comment:
You probably forgot to add `script_callbacks` module to this PR.

Reply:
the missed file was added